pkg/ui/package.json

```diff
@@ -44,13 +38,16 @@
 		"typescript": "^4.7.4",
 		"vite": "^3.0.0",
 		"vitest": "~0.21.1",
-		"storybook-addon-pseudo-states": "~1.9.0",
 		"svelte2tsx": "~0.5.13",
-		"chromatic": "~6.7.4",
-		"@types/uuid": "~8.3.4"
+		"@types/uuid": "~8.3.4",
+		"histoire": "~0.11.2",
+		"@histoire/plugin-svelte": "~0.11.2",
+		"@histoire/controls": "~0.11.3",
+		"autoprefixer": "~10.4.12"
 	},
 	"type": "module",
 	"dependencies": {
-		"uuid": "~9.0.0"
+		"uuid": "~9.0.0",
+		"lucide-svelte": "~0.91.0"
```

Comment:
I would move `lucide-svelte` from `dependencies` and add it to both `peerDependencies` and `devDependencies` + externalise it when bundling with Vite. 

The reasoning is: the library shouldn't be packaged with the UI as it will still be loaded when packaging `web-client` (and we're using the UI package solely for `web-client` and not publishing it, so there's no need to bundle the svg's in)

Reply:
Thanks for this! Never would have thought about it and its v. important.